lighthouse-core/gather/driver.js

```diff
@@ -529,7 +529,10 @@ class Driver {
     const response = await this.sendCommand('Runtime.evaluate', evaluationParams);
     if (response.exceptionDetails) {
       // An error occurred before we could even create a Promise, should be *very* rare
-      return Promise.reject(new Error(`Evaluation exception: ${response.exceptionDetails.text}`));
+      const errorMessage = response.exceptionDetails.exception ?
+        response.exceptionDetails.exception.description :
```

Comment:
did you run into this?

Reply:
I sure did. If the JS you give to evaluateAsync is malformed, it errors in such a way that triggers this code. And `response.exceptionDetails.text` was just `Uncaught`! the real error was in the desc.

Reply:
Granted this is totally unrelated to this change, I came across it during implementation.